docs/pages/blog/2021-q2-update.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Q2 2021 Update
+description: An update on our mission for Q2 2021.
+date: 2021-07-12T00:00:00.000Z
+authors: ['oliviertassinari']
+card: true
+---
+
+This update covers our progress over the last three months and what we aim to achieve in the months ahead.
```

Comment:
```suggestion
This update covers our progress over the last three months, and what we aim to achieve in the months ahead.
```

Reply:
I was following https://www.grammarly.com/blog/comma-in-complex-sentences/.